docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,23 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Tempo datasource: "Aggregate By" Search option to compute RED metrics over spans aggregated by attribute
+
+<!-- Joey Tawadrous, Jen Villa -->
+<!-- available in on-prem starting with Grafana 10.2, but also requires Tempo/GET v2.2 or greater -->
+
+October 24, 2023
+
+_Experimental in Grafana Cloud_
+
+We've added an "Aggregate By" option to the [TraceQL query editor](https://grafana.com/docs/grafana/next/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Grafana Cloud Traces' [metrics summary API](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/). Users can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
```

Comment:
Your link actually works just fine, I'm just using version interpolation to simplify them.

```suggestion
We've added an **Aggregate By** option to the [TraceQL query editor](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/query-editor/traceql-search/#write-traceql-queries-using-search) to leverage Grafana Cloud Traces' [metrics summary API](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/traces/metrics-summary-api/). You can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` received in the last hour that match your filter criteria, grouped by whatever attributes you specify.
```